Services/AuthenticationService/Controllers/AuthenticationController.cs

```diff
@@ -27,11 +30,11 @@ public AuthenticationController([FromServices] ITokensEngine tokenEngine)
         /// </summary>
         [Route("login")]
         [HttpPost]
-        public string Login([FromServices] ILoginCommand command, [FromBody] UserEmailPassword user)
+        public async Task<string> Login([FromServices] ILoginCommand command, [FromBody] UserEmailPassword user)

```

Comment:
I don't understand why it is async?

Reply:
We have to make async all calls to not stop services work, as I understood

Reply:
> I don't understand why it is async?

Because there's asynchronous backend logic